test/e2e/discoveryconfig_controller.go

```diff
@@ -259,14 +274,14 @@ func getTestserverDeployment() *appsv1.Deployment {
 	testserverDeployment := &appsv1.Deployment{}
 	Eventually(func() error {
 		return k8sClient.Get(ctx, testserver, testserverDeployment)
-	}, timeout, interval).ShouldNot(HaveOccurred())
+	}, timeout, interval).ShouldNot(HaveOccurred(), "Namespace "+discoveryNamespace)
```

Comment:
Instead of concat strings with + operator, perhaps use `fmt.sprintf("Namespace %s", discoveryNamespace)`?

Reply:
I'll actually just take that out. I was using the log to debug while getting the configurable namespace to work, so it's not necessary